homework-2/src/components/Menu.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import MenuItem from './MenuItem.jsx';
+
```

Comment:
```suggestion
import PropTypes from 'prop-types'
```
так как вы тут используете PropTypes